netutils/webserver/httpd_dirlist.c

```diff
@@ -193,11 +193,8 @@ ssize_t httpd_dirlist(int outfd, FAR struct httpd_fs_file *file)
           break;
         }
 
-      path = malloc(CONFIG_NAME_MAX);
-      ASSERT(path);
-
-      snprintf(path, CONFIG_NAME_MAX, "%s/%s",
-               file->path, dent->d_name);
+      ret = asprintf(&path, "%s/%s", file->path, dent->d_name);
+      ASSERT(ret >= 0 && path);
```

Comment:
```suggestion
      ASSERT(ret > 0 && path);
```
so at least "/" should be printed

Reply:
Done.